starter_code/src/movies.js

```diff
@@ -1,20 +1,105 @@
 /* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
+// Turn duration of the movies from hours to minutes
 
+function turnHoursToMinutes(movieArray) {
```

Comment:
Anoop- I explained to Bastien on how to do this. I can explain on how to make Jasmine happy during a break if you'd like. Or I can explain it on Monday if you want to have a go at finishing it over the weekend :)